qiskit_nature/drivers/molecule.py

```diff
@@ -39,6 +39,7 @@ def __init__(
         geometry: List[Tuple[str, List[float]]],
         multiplicity: int = 1,
         charge: int = 0,
+        unit: str = "Angstrom",
```

Comment:
I suggest you use the `UnitsType` enum which is being imported anyways.

Reply:
I tried using UnitsType.ANGSTROM but its not resolving to Angstrom
```
print(UnitsType.ANGSTROM)
UnitsType.ANGSTROM
```
I am getting this.